website/README.md

```diff
@@ -18,6 +18,13 @@ yarn start
 
 You will then be able to open http://localhost:3000 in your browser.
 
+To clean any generated files, you can run:
+
+```
+yarn clear
+rm -rf node_modules
+```
+
```

Comment:
Anyone have a better way of doing this?

Reply:
`git clean`?

Reply:
`git clean -ffdx` will completely obliterate anything not tracked by git and restore the directory to what it would look like from a fresh `git clone`

Reply:
Why is this needed?

Reply:
When I built the site, it added a whole bunch of files that were messing with my code search, so I wanted an easy way to delete them.

Reply:
Easier to ignore them in your IDE, usually :)

Reply:
OK. I'll remove this.